javascript/builtins/Temporal/PlainDate.json

```diff
@@ -1690,7 +1690,9 @@
                 },
                 "edge": "mirror",
                 "firefox": {
-                  "version_added": "139"
+                  "version_added": "139",
+                  "partial_implementation": true,
+                  "notes": "The following calendars are not yet support `islamic`, `islamic-rgsa` and `islamic-umalqura`. See [bug 1950425](https://bugzil.la/1950425)."
```

Comment:
```suggestion
                  "notes": "Calendars `chinese`, `dangi`, `islamic`, `islamic-rgsa` and `islamic-umalqura` are not available. See [bug 1950425](https://bugzil.la/1950425)."
```

Reply:
I have test every method and this the only one that is affected.
I have also tested every calendar type and the only ones that don't work are the ones that I mentioned.
[Comment 4](https://bugzilla.mozilla.org/show_bug.cgi?id=1950425#c4) explains where these were corrrected co the name of the bug is the issue here.

Reply:
Ah, looks like https://github.com/mozilla/gecko-dev/commit/4618744e6667072b2eed0b9757bca7d59278cde5 as part of https://bugzilla.mozilla.org/show_bug.cgi?id=1954138 is the explanation:

> Bug 1954138 - Part 9: Re-enable Chinese and Dangi calendars even when not Nightly.